.github/CONTRIBUTING.md

```diff
@@ -49,6 +65,11 @@ outlined on that page and do not file a public issue.
 * 4 spaces for indentation rather than tabs
 * Public classes and methods must contain header documentation
 * Use plain C functions whenever possible (as opposed to class methods)
+* Restrict subclassing (objc macro, final in examples)
+* ivars v props
+* local vars instead of self.prop over & over
```

Comment:
Hey @jessesquires, thanks for mentioning out the changes :) I fixed the things related to your 3 comments and now I will commit it again :) Let me know what you think.